test/dattri/algorithm/test_tracin.py

```diff
@@ -30,6 +30,7 @@ def test_tracin_proj(self):
             torch.randint(0, 10, (10,)),
         )
         train_loader = DataLoader(train_dataset, batch_size=4)
+        test_loader = train_loader
```

Comment:
No need to change this file since the self influence will not be applied to RPS.